src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs

```diff
@@ -167,6 +167,17 @@ public async Task<IActionResult> Create(CreateOpenIdApplicationViewModel model,
                 Type = model.Type
             };
 
+            var openIdServerSettings = await GetServerSettingsAsync();
```

Comment:
While I agree this must be fixed, I don't think it's the right thing to do. Instead, we should consider adding a new checkbox to allow clients to use the logout endpoint.

Reply:
Do we still need this call?

Reply:
Nope, removed it